plugins/woocommerce/includes/class-wc-discounts.php

```diff
@@ -333,7 +333,7 @@ protected function get_items_to_apply_coupon( $coupon ) {
 
 			$items_to_apply[] = $item_to_apply;
 		}
-		return $items_to_apply;
+		return apply_filters( 'woocommerce_coupon_get_items_to_apply', $items_to_apply, $coupon, $this );
```

Comment:
This needs docblock. Please review https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/ for guidance.

Reply:
Thanks @budzanowski . I added the missing docbloc here: ace39fa73faffff26d75563fdb79d45decaea1f2